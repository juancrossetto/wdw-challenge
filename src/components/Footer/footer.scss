@use '../../scss/' as *;

.footer {
  height: 61px;
  width: 100%;
  background-color: $dark;
  color: $white;
  border-top: 6px solid $primary;
  pointer-events: none;

  &__inner {
    @include flex(center, center);
    height: 100%;
  }

  &__logo {
    width: 145px;
    pointer-events: visible;
    margin-right: 14px;
    display: flex;
    @include for-phone-only {
      display: none;
    }
    img {
      width: 100%;
    }
  }

  &__text {
    margin: 0;
    font-family: 'Nunito Sans', sans-serif;
    font-style: normal;
    font-weight: 400;
    font-size: 19px;
    line-height: 26px;
    @include for-phone-only {
      font-weight: 600;
      text-align: center;
    }
  }
}
