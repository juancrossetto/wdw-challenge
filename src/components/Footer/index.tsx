import React from 'react';
import { logoWhite } from '../../assets/images';
import { Button } from 'react-bootstrap';
import './footer.scss';

const Footer = () => {
  return (
    <div className="footer">
      <div className="footer__inner container">
        <div className="footer__logo">
          <img src={logoWhite} alt="" />
        </div>
        <p className="footer__text">Â© All rights reserved {new Date().getFullYear()}</p>
      </div>
    </div>
  );
};

export default Footer;
