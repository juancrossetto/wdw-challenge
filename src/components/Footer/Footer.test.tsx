import React from 'react';
import renderer from 'react-test-renderer';

import Footer from '.';

describe('Footer component', () => {
  test('Footer render correctly', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
