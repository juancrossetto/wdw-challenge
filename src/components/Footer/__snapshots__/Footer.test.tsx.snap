// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Footer render correctly 1`] = `
<div
  className="footer"
>
  <div
    className="footer__inner container"
  >
    <div
      className="footer__logo"
    >
      <img
        alt=""
        src={
          Object {
            "process": [Function],
          }
        }
      />
    </div>
    <p
      className="footer__text"
    >
      Â© All rights reserved 
      2022
    </p>
  </div>
</div>
`;
